site/content/integrate/apps/functionality/debug.md

```diff
@@ -0,0 +1,56 @@
+---
+title: "Debug Apps"
+heading: "Debug Apps"
+description: "`/apps debug` command can be used to debug an App's bindings, KV store records, view the logs, etc."
+weight: 50
+---
+The Apps framework includes a built-in slash command, `/apps debug`, that enables App debugging features, such as viewing logs and listing key/value (K/V) store records.
+
+{{<note>}}
+App debug features require the **Enable Developer Mode** and **Enable Testing Commands** settings be set to `true` in the System Console. Both settings can be located in the **Environment > Developer** section.
+{{</note>}}
```

Comment:
That is no longer true. Since https://github.com/mattermost/mattermost-plugin-apps/pull/372 Development mode is enabled via `/apps settings`.

Reply:
Not, https://github.com/mattermost/mattermost-plugin-apps/pull/372 is not released yet. It will be in the same release as https://github.com/mattermost/mattermost-plugin-apps/pull/394.

Reply:
@neflyte this content is fine, but should not be pushed to production until Apps Framework 1.2.  This should be in the next 1-2 weeks.

Reply:
@hanzei, I've been told this will be released in 1.2 in the next 1-2 weeks.  We should hold this until then, but

Reply:
0/5 I don't see why not - let's just get them both merged quickly.

Reply:
Updated the docs in https://github.com/mattermost/mattermost-developer-documentation/pull/1211